diff --git a/docs/tutorials/train_painn_qm9.py b/docs/tutorials/train_painn_qm9.py
index d962025..00b40cf 100644
--- a/docs/tutorials/train_painn_qm9.py
+++ b/docs/tutorials/train_painn_qm9.py
@@ -92,8 +92,8 @@ inference_model = property + 'best_inference_model'
 cutoff = args.cutoff
 n_atom_basis = args.nf
 
-if property == "r2" or "mu" or "alpha" or "cv":
-    qm9data = QM9(
+
+qm9data = QM9(
     os.path.join(qm9tut, 'qm9' + property + '.db'),
     batch_size=args.batch_size,
     num_train=100000,
@@ -103,28 +103,12 @@ if property == "r2" or "mu" or "alpha" or "cv":
         trn.RemoveOffsets(QM9.property, remove_mean=True, remove_atomrefs=args.remove_atomrefs),
         trn.CastTo32()
     ],
+    property_units= None if property == "r2" or "mu" or "alpha" or "cv" else {QM9.property: 'eV'},
     num_workers=args.num_workers,
     split_file=os.path.join(qm9tut, "split.npz"),
     pin_memory=True, # set to false, when not using a GPU
     load_properties=[QM9.property], #only load homo property
     remove_uncharacterized = True)
-else:
-    qm9data = QM9(
-        os.path.join(qm9tut, 'qm9' + property + '.db'),
-        batch_size=args.batch_size,
-        num_train=100000,
-        num_val=17748,
-        transforms=[
-            trn.ASENeighborList(cutoff=args.cutoff),
-            trn.RemoveOffsets(QM9.property, remove_mean=True, remove_atomrefs=args.remove_atomrefs),
-            trn.CastTo32()
-        ],
-        property_units={QM9.property: 'eV'},
-        num_workers=args.num_workers,
-        split_file=os.path.join(qm9tut, "split.npz"),
-        pin_memory=True, # set to false, when not using a GPU
-        load_properties=[QM9.property], #only load homo property
-        remove_uncharacterized = True)
 
 qm9data.prepare_data()
 qm9data.setup()
